pulsar-broker/src/main/java/com/yahoo/pulsar/broker/namespace/OwnershipCache.java

```diff
@@ -130,9 +127,16 @@
                             if (LOG.isDebugEnabled()) {
                                 LOG.debug("Successfully acquired zk lock on {}", namespaceBundleZNode);
                             }
+                            final OwnedBundle bundle = new OwnedBundle(
+                                    ServiceUnitZkUtils.suBundleFromPath(namespaceBundleZNode, bundleFactory));
+                            // update the version
+                            localZkCache.getZooKeeper().exists(path, false, (rc1, path1, ctx1, stat) -> {
```

Comment:
when it gets removed from ZK it also gets removed from the `OwnershipCache`, broker-2 is not supposed to own the bundle at the same time as anyone else 😄